src/limits-config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
may be rename to `limits-severity-css-class-mapping.json`

Reply:
It's a color mapping and I want it to be more generic in case we change it in the future. I suggest we leave it as is.